File: src/main/java/com/zunza/buythedip/crypto/repository/CryptoMetadataRepository.java
package com.zunza.buythedip.crypto.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.zunza.buythedip.crypto.entity.CryptoMetadata;

@Repository
public interface CryptoMetadataRepository extends JpaRepository<CryptoMetadata, Long> {
	Optional<CryptoMetadata> findByCryptoId(Long cryptoId);
}

File: src/main/kotlin/com/zunza/buythedip/crypto/repository/CryptoMetadataRepository.kt
package com.zunza.buythedip.crypto.repository

import com.zunza.buythedip.crypto.entity.CryptoMetadata
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CryptoMetadataRepository : JpaRepository<CryptoMetadata, Long> {
}
